grails-app/services/au/org/ala/downloads/LoggerService.groovy

```diff
@@ -97,6 +98,11 @@ class LoggerService {
         final port = grailsApplication.config.app.logger.port ?: '80'
         final path = grailsApplication.config.app.logger.path ?: '/service/logger/'
         final reasons = grailsApplication.config.app.logger.reasons ?: 'reasons'
-        httpWebService.getJson("${server}:${port}${path}${reasons}")
+        def resp = webService.get("${server}:${port}${path}${reasons}")
+        if (resp.error) {
```

Comment:
Looking at WebService.groovy send(), which post() and get() will call, it populates result.error:


       try {
            url = appendQueryString(url, params)

            HTTPBuilder http = newHttpBuilder(url, contentType)

            http.request(method, contentType) { request ->
                configureRequestTimeouts(request)
                configureRequestHeaders(headers, includeApiKey, includeUser, customHeaders)

                if (files != null) {
                    // NOTE: order is important - Content-Type MUST be set BEFORE the body
                    request.entity = constructMultiPartEntity(body, files, contentType)
                } else if (body != null) {
                    // NOTE: order is important - Content-Type MUST be set BEFORE the body
                    delegate.contentType = contentType
                    delegate.body = body
                }

                response.success = { resp, data ->
                    result.statusCode = resp.status
                    if (data instanceof InputStreamReader) {
                        result.resp = data.text
                    } else if (data instanceof List) {
                        // ensure an empty list is not converted to an empty object
                        result.resp = data
                    } else {
                        result.resp = data ?: [:]
                    }
                }
                response.failure = { resp ->
                    log.error("Request failed with response: ${resp?.entity?.content?.text}")
                    result.statusCode = resp.status
                    result.error = "Failed calling web service - service returned HTTP ${resp.status}"
                }
            }